src/etc/inc/config.inc

```diff
@@ -49,7 +49,11 @@ final class product
 
         /* boot detection */
         $bootflag = '/var/run/booting';
-        @touch($bootflag);
+
+        if (! file_exists($bootflag)) {
```

Comment:
might not pick it up indeed... the style is (or at least that is what we use everywhere)

```suggestion
        if (!file_exists($bootflag)) {
```

Reply:
Thanks, I did open a PR to the stable version: https://github.com/opnsense/core/pull/6604

Do I need to include it here too ?
Or do you merge stable into dev ?

Reply:
yes, master please. Sorry for the trouble. Could commit this myself but in the open source spirit external authors are welcome and encouraged :)

Reply:
Well, I force pushed this PR. Still have to fix the comment about ping hosts. But you can probably cherry-pick my commit for the booting file, I will rebase ;)

Reply:
ok done :) 97343e3aae

Reply:
Awesome, thank you !
Will it go to the stable release, is there a procedure ?